buildconfig/CMake/FindPyQt.py

```diff
@@ -35,12 +35,12 @@ def __init__(self, name):
           self.sip_dir = os.path.join(sys.prefix, 'sip', name)
       elif sys.platform == 'darwin':
           # hardcoded to homebrew Cellar
-          cellar_prefix = '/usr/local/Cellar'
+          cellar_prefix = '/usr/local/opt'
```

Comment:
Is this variable name and the above comment now wrong to refer to Cellar?

Reply:
Everything under `opt` is symlinked to the `Cellar` just without the version so I think we can still justify it.